.taskcluster.yml

```diff
@@ -82,7 +82,7 @@ tasks:
               /repo/bin/configure_ssh.sh &&
               node /repo/bin/get_secret.js &&
               cd mozilla-central &&
-              ./mach try -p linux64 -u all -t none --artifact
+              ./mach try -b o -p linux64 testing/extensions -t tp6-e10s --artifact
```

Comment:
Hm. Does `-t all` work? Are you sure that artifact builds work with Talos?

ISTR they do but reading https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Artifact_builds: 

> "Tests other than mochitests or xpcshell tests. There aren't inherent barriers here, but these are not known to work."

Reply:
I'll try locally.